docs/federation/types.md

```diff
@@ -85,6 +85,12 @@ An [ISO 8601][iso8601] date.
 
 Example: `2016-02-19`
 
+## Timezone
+
+A timezone in the form `Area/Location` as used in the [Time Zone Database][tz].
```

Comment:
The timestamp is stored and federated in UTC. This is used to display the time in this timezone, and if not set it is displayed in the timezone of the viewer.

That is useful if you have a local event (birthday party at your home) and a friend from another country/timezone sees this event always with the timezone you set (the timezone of your home).

If not set, it's displayed in the timezone of the viewer, useful for a livestream or something like that, so you see when the event is in your timezone.

I don't know if it is reliable to use the location for that, I think it is better to set the timezone explicitly instead of doing some location magic.